crates/types/src/errors.rs

```diff
@@ -166,6 +166,7 @@ pub mod codes {
         INTERNAL 500 "Internal",
         ABORTED 409 "Aborted",
         GONE 410 "Gone",
+        UNSUPPORTED_MEDIA_TYPE 415 "Unsupported media type",
```

Comment:
Wondering whether unsupported media type would be as understandable as incompatible/unsupported service protocol version. Can we instead say `Incompatible service protocol version` and make it a 5xx?

Reply:
for now no, this is the status code we get from the SDK, and my goal here was to align those two to avoid having 2 logs messages in service and runtime with different codes.
It also prints a good message already in the SDK and in the runtime, with even a link to the doc page that tells you what to do (in the UI the doc page is rendered in a tooltip, so even nicer), so really no need to add more i think.

Reply:
there is a larger problem here is that this should be using potentially another status code, that cannot conflict with status codes the user might be already using... But not gonna change this for now, will keep it like that.